app/src/main/java/nerd/tuxmobil/fahrplan/congress/settings/SettingsFragment.kt

```diff
@@ -92,4 +147,10 @@ class SettingsFragment : PreferenceFragment() {
         val intent = Intent(Settings.ACTION_APP_NOTIFICATION_SETTINGS).apply { putExtras(extras) }
         startActivity(intent)
     }
+
+    /**
+     * See [findPreference].
```

Comment:
This comment feels redundant to me. The name of the function is clear enough. If readers need more information the method body is more useful/clear than the comment.

Reply:
:heavy_check_mark: Updated in 23b10ca4822a133ed18915842987e132b4a9205a.